google/cloud/bigtable/examples/read_snippets.cc

```diff
@@ -121,8 +121,8 @@ void ReadRowsWithLimit(google::cloud::bigtable::Table table,
         cbt::Filter::ColumnRangeClosed("stats_summary", "connected_wifi",
                                        "connected_wifi"),
         cbt::Filter::Latest(1));
-    // Read and print the first 5 rows in the range.
-    for (StatusOr<cbt::Row> const& row : table.ReadRows(range, 5, filter)) {
+    // Read and print the first rows in the range, within the row limit.
```

Comment:
I think this example is shared with other libraries, check with @billyjacobson to make sure we are not making the example harder to follow in C++ vs. other languages.

Reply:
This example actually doesn't have a DevRel region tag. It only has a tag for our dox.

You may be thinking of this [Limit Cells Per Row example](https://cloud.google.com/bigtable/docs/samples/bigtable-filters-limit-cells-per-row) found in `data_filter_snippets.cc`. (why are there so many different files -- I do not know.)

I'm also noticing that the filter example I linked does not look like the other languages' examples... (I will fix)

Reply:
Actually none of the filter examples conform to what other libraries use. I opened #6868